script/main.py

```diff
@@ -129,122 +133,83 @@ def create_users_table(conn):
 
 
 def insert_users_to_table(conn, users):
-  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
+  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
 
   cur = conn.cursor()
   for columns in users:
     cur.execute(sql, columns)  
   conn.commit()
 
 
+def insert_users_to_db(conn, people):
+  users = []
+
+  peoples = people['results']
+  for dict_of_person in peoples:      
+    fields = [
+      dict_of_person['gender'],
+      dict_of_person['name']['title'],
+      dict_of_person['name']['first'],
+      dict_of_person['name']['last'],
+      dict_of_person['location']['street']['number'],
+      dict_of_person['location']['street']['name'],
+      dict_of_person['location']['city'],
+      dict_of_person['location']['state'],
+      dict_of_person['location']['country'],
+      dict_of_person['location']['postcode'],
+      dict_of_person['location']['coordinates']['latitude'],
+      dict_of_person['location']['coordinates']['longitude'],
+      dict_of_person['location']['timezone']['offset'],
+      dict_of_person['location']['timezone']['description'],
+      dict_of_person['email'],
+      dict_of_person['login']['uuid'],
+      dict_of_person['login']['username'],
+      dict_of_person['login']['password'],
+      dict_of_person['login']['salt'],
+      dict_of_person['login']['md5'],
+      dict_of_person['login']['sha1'],
+      dict_of_person['login']['sha256'],
+      dict_of_person['dob']['date'],
+      dict_of_person['dob']['age'],
+      dict_of_person['dob']['time_until_birthday'],
+      dict_of_person['registered']['date'],
+      dict_of_person['registered']['age'],
+      dict_of_person['phone'],
+      dict_of_person['cell'],
+      dict_of_person['id']['name'],
+      dict_of_person['id']['value'],
+      dict_of_person['nat'],
+    ]
+
+    users.append(fields)      
+    
+  insert_users_to_table(conn, users)
+
+
 def init_db(conn):  
-  '''Initializing the database'''    
+  people =  json.load(open("init\persons.json", encoding='utf-8'))
+  (phone_fields, dob_fields) = convert_dict_to_list_access_phone_cell_dob_field(people)
+  add_field_time_until_birthday(dob_fields)
+  remove_special_characters_from_phone_numbers(phone_fields)
+  remove_field_with_picture_from_records(people)   
```

Comment:
What's the point of calling this function here?
No matter if you remove the pictures or not. The database will look the same.

Reply:
I have to call this function somewhere. If main () should be clean, the next place it should be is the init () function.
When I'm wrong, correct me.

Reply:
No. The question was: Is this function doing anything useful?
It doesn't matter if you remove pictures or not. You will not add them to the database anyway

Reply:
In fact, I didn't think about its meaning. The command was to delete it, I deleted it.